### ISSUE TITLE
Angular kendo-menu doesn't like Bootstrap 4.2.1

### ISSUE THREAD

**Issue by jcastillo-ait-asia** [telerik/kendo-themes#579]
## I'm submitting a...

* Bug report 

## Current behavior

Getting the following error on build:

```
ERROR in ./src/styles.scss
Module build failed (from /Users/johnk/git/uber/uber-spa/node_modules/mini-css-extract-plugin/dist/loader.js):
ModuleBuildError: Module build failed (from /Users/johnk/git/uber/uber-spa/node_modules/sass-loader/lib/loader.js):

$menu-text: rgba( $header-text, .5 ) !default;
           ^
      Argument `$color` of `rgba($color, $alpha)` must be a color
      in /Users/johnk/git/uber/uber-spa/node_modules/@progress/kendo-theme-bootstrap/scss/menu/_theme.scss (line 2, column 13)
    at runLoaders (/Users/johnk/git/uber/uber-spa/node_modules/webpack/lib/NormalModule.js:286:20)
    at /Users/johnk/git/uber/uber-spa/node_modules/loader-runner/lib/LoaderRunner.js:364:11
    at /Users/johnk/git/uber/uber-spa/node_modules/loader-runner/lib/LoaderRunner.js:230:18
    at context.callback (/Users/johnk/git/uber/uber-spa/node_modules/loader-runner/lib/LoaderRunner.js:111:13)
    at Object.render [as callback] (/Users/johnk/git/uber/uber-spa/node_modules/sass-loader/lib/loader.js:52:13)
    at Object.done [as callback] (/Users/johnk/git/uber/uber-spa/node_modules/neo-async/async.js:8077:18)
    at options.error (/Users/johnk/git/uber/uber-spa/node_modules/node-sass/lib/index.js:294:32)
```

**Comment by elena-gancheva** [comment]
Hi @jcastillo-ait-asia could you please provide a screenshot of how you import your `scss` files? It looks like the importing order is causing this problem.

**Comment by JeroenRombouts** [comment]
We seem to have the exact same problem (with xxx replacing the path to node_modules, because it identifies us)

ERROR in xxx/libs/styles/src/styles.scss
Module build failed: ModuleBuildError: Module build failed:
undefined
           ^
      Argument `$color` of `rgba($color, $alpha)` must be a color
      in xxx\node_modules\@progress\kendo-theme-bootstrap\scss\menu\_theme.scss (line 2, column 13)
    at runLoaders (xxx\node_modules\webpack\lib\NormalModule.js:252:20)
    at xxx\node_modules\loader-runner\lib\LoaderRunner.js:367:11
    at xxx\node_modules\loader-runner\lib\LoaderRunner.js:233:18
    at context.callback (xxx\node_modules\loader-runner\lib\LoaderRunner.js:111:13)
    at Object.render [as callback] (xxx\node_modules\sass-loader\lib\loader.js:52:13)
    at Object.done [as callback] (xxx\node_modules\neo-async\async.js:8077:18)
    at options.error (xxx\node_modules\node-sass\lib\index.js:294:32)

**Comment by jcastillo-ait-asia** [comment]
> Hi @jcastillo-ait-asia could you please provide a screenshot of how you import your `scss` files? It looks like the importing order is causing this problem.

Hello @elena-gancheva, this is our import in ```styles.scss``` file:

```
@import 'variables';
@import '../../../node_modules/npm-font-open-sans/open-sans.scss';
@import '../../../node_modules/bootstrap/scss/bootstrap.scss';
@import '../../../node_modules/bootstrap/scss/bootstrap-grid.scss';
@import '../../../node_modules/bootstrap/scss/bootstrap-reboot.scss';
@import '../../../node_modules/@ngx-share/button/themes/outline/outline-theme';
@import '../../../node_modules/@progress/kendo-theme-bootstrap/scss/calendar';
@import '../../../node_modules/@progress/kendo-theme-bootstrap/scss/datetime';
@import '../../../node_modules/@progress/kendo-theme-bootstrap/scss/listbox';
@import '../../../node_modules/@progress/kendo-theme-bootstrap/scss/dialog';
@import '../../../node_modules/@progress/kendo-theme-bootstrap/scss/window';
@import '../../../node_modules/@progress/kendo-theme-bootstrap/scss/dropdownlist';
@import '../../../node_modules/@progress/kendo-theme-bootstrap/scss/dropdowngrid';
@import '../../../node_modules/@progress/kendo-theme-bootstrap/scss/dropdowntree';
@import '../../../node_modules/@progress/kendo-theme-bootstrap/scss/multiselect';
@import '../../../node_modules/@progress/kendo-theme-bootstrap/scss/switch';
@import '../../../node_modules/@progress/kendo-theme-bootstrap/scss/slider';
@import '../../../node_modules/@progress/kendo-theme-bootstrap/scss/dataviz';
@import '../../../node_modules/@progress/kendo-theme-bootstrap/scss/grid';
@import '../../../node_modules/@progress/kendo-theme-bootstrap/scss/upload';
@import '../../../node_modules/@progress/kendo-theme-bootstrap/scss/autocomplete';
@import '../../../node_modules/@progress/kendo-theme-bootstrap/scss/menu';
@import '../../../node_modules/@progress/kendo-theme-bootstrap/scss/scheduler';
@import '../../../node_modules/@progress/kendo-theme-bootstrap/scss/button';
@import '../../../node_modules/@progress/kendo-theme-bootstrap/scss/popup';
@import '../../../node_modules/@progress/kendo-theme-bootstrap/scss/input';
```

**Comment by JeroenRombouts** [comment]
Fyi: our problem went away when I downgraded Bootstrap to version 4.1.x.

**Comment by joneff** [comment]
The issue is related to the [usage of `null` values in Bootstrap 4.2.x](https://github.com/twbs/bootstrap/blob/v4-dev/scss/_variables.scss#L822). We haven't yet upgraded kendo-theme-bootstrap to support it, hence the issue.

We'll try to upgrade as soon as possible.

**Comment by merobal** [comment]
Related: https://github.com/telerik/kendo-themes/issues/520

**Comment by joneff** [comment]
Fixed by twbs/boostrap@5adcfe97f04a5c2f0b2d0712a6f2332c0bbd8446. Note: you don't need to update kendo-theme-bootstrap. Just bootstrap.

**Comment by joneff** [comment]
Just a minor clarification, the issue was not caused by the `null` value. it was caused by the `inherit` value.

The way we, and bootstrap for that matter, have organized our files a `null` value allows for a later assignment. When the value is set to `inherit` it does not allow said assignment.
